README.md

```diff
@@ -1,12 +1,21 @@
 # PAWS
 
-PAWS: A Web Shell (PAWS) is a Jupyter notebooks deployment that has been customized to make interacting with Wikimedia wikis easier. It allows users to create and share documents that contain live code, visualizations such as graphs, rich text, etc. The user created notebooks are a powerful tool that enables data analysis and scientific research, and also transforms the way in which programmers write code - by enabling an exploratory environment with a quick feedback loop, and a low barrier for entry through it's easy to use graphical interface. 
-
-How to contribute? Just simply clone on your computer by writing the following command: 
-
-` git clone https://github.com/toolforge/paws.git `
-
-`cd paws`
+[PAWS](https://www.mediawiki.org/wiki/PAWS): A Web Shell (PAWS) is a Jupyter notebooks deployment that has been customized to make interacting with Wikimedia wikis easier. It allows users to create and share documents that contain live code, visualizations such as graphs, rich text, etc. The user created notebooks are a powerful tool that enables data analysis and scientific research, and also transforms the way in which programmers write code - by enabling an exploratory environment with a quick feedback loop, and a low barrier for entry through it's easy to use graphical interface. 
+
+It is a customized JupyterHub on Kubernetes running inside Wikimedia Cloud Services. It provides a simple way to use Python and pywikibot interact with the Wikimedia Projects, including the various wikis via OAuth, the Wiki Replicas Cloud service, and others. 
```

Comment:
Please reword this sentence to make more sense.

Specifically: 'It provides a simple way to use Python and pywikibot to interact with the wikimedia projects [..]" (note missing 'to'.)